src/main/java/domain/piece/AbstractPiece.java

```diff
@@ -0,0 +1,16 @@
+package domain.piece;
+
+import domain.Team;
+
+public abstract class AbstractPiece implements Piece {
+
+    protected final Team team;
+
+    protected AbstractPiece(final Team team) {
+        this.team = team;
+    }
+
+    public final boolean isGreenTeam() {
+        return Team.isGreenTeam(team);
```

Comment:
```suggestion
        return team.isGreenTeam();
```

정적 메서드를 이용할 필요가 없는 것 같네요

Reply:
감사합니다.  
다시봐도 필요없을 것 같습니다.  
제안해주신 내용대로 수정했습니다.